addons/spreadsheet/static/src/list/plugins/list_ui_plugin.js

```diff
@@ -286,6 +297,21 @@ export class ListUIPlugin extends OdooUIPlugin {
         }
     }
 
+    _addTable({ sheetId, col, row, linesNumber, columns }) {
+        const zone = {
+            left: col,
+            right: col + columns.length - 1,
+            top: row,
+            bottom: row + linesNumber,
+        };
+        this.dispatch("CREATE_TABLE", {
+            tableType: "static",
+            sheetId,
+            ranges: [this.getters.getRangeDataFromZone(sheetId, zone)],
+            config: { ...PIVOT_TABLE_CONFIG },
```

Comment:
style should be changed to have `firstColumn: false`

Reply:
-> updated with a test step